test/avi/test_helpers.clj

```diff
@@ -140,6 +140,25 @@
   [expected]
   (line nil expected))
 
+(defn terminal-buffer
```

Comment:
I have made some changes related to the same and pushed the changes. Have a look and give comments.